subsys/ipc/ipc_service/backends/ipc_icbmsg.c

```diff
@@ -1087,7 +1087,6 @@ static int register_ept(const struct device *instance, void **token,
 	struct ept_data *ept = NULL;
 	bool matching_state;
 	int ept_index;
-	int r = 0;
```

Comment:
The `r` is never changed, so what's the point of creating this variable?

Reply:
Exactly - this is why I am removing it, not adding.